# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d9/ShaderCache.h
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d9/ShaderCache.h
--- a/gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d9/ShaderCache.h	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d9/ShaderCache.h	Sat Sep 07 13:38:36 2019 +0000
@@ -36,7 +36,7 @@
 
     void initialize(IDirect3DDevice9 *device) { mDevice = device; }
 
-    angle::Result create(Context9 *context9,
+    angle::Result create(d3d::Context *context,
                          const DWORD *function,
                          size_t length,
                          ShaderObject **outShaderObject)
@@ -54,7 +54,7 @@
 
         ShaderObject *shader;
         HRESULT result = createShader(function, &shader);
-        ANGLE_TRY_HR(context9, result, "Failed to create shader");
+        ANGLE_TRY_HR(context, result, "Failed to create shader");
 
         // Random eviction policy.
         if (mMap.size() >= kMaxMapSize)